### PR TITLE
ATO-1077: omitted claims

### PR DESCRIPTION
**Pull request by Ryan-Andrews99**
## What:
- The IPV stub now only renders form inputs for claims included in the request object (ie you can't return things for claims that were not requested)
- This applies to all but the coreIdentity and return code claims (as there are scenarios in which we still want a return code but not include the claim)
- Adds form fields for sub, vtr and vot claims, the last two of which are editable
- The authorize endpoint can now handle empty/undefined form fields without throwing a JSON error 
- Updates the rendering of the request claims to be a bit nicer
- Some BAU stuff which have their own commits with explanations
## How to review:
- Code review commit-by commit 
- Pull the stub down and run locally, use the request object helper script to vary the claims included, observe conditional rendering of some  fields for the form claims 
- Deployed to dev and checked all works fine

### CODE REVIEW

File: src/main/ipv-stub/README.md [hunk 1]
```diff
@@ -31,3 +31,11 @@ npm run createRequestObject
...
 
 This will use the local keys in the `parameters.json` file to create a valid signed and encrypted request object. You can then run through the stub and be redirected to `https://oidc.local.account.gov.uk/ipv-callback` with an auth code
+
+If you are developing locally with the stub, you can automatically rebuild the stub by running the following command:
+
+```bash
+npm run build:ipv:watch
+```
+
+In a separate terminal, run the command `npm run start:local:ipv` to start the stub. Sam local will automatically reload changes when the `build/` directory is updated, meaning changes are hot reloaded.
```
Comment (BenjaminWCO) [inline src/main/ipv-stub/README.md:41]: We need to remember to make sure we increase the timeout if this causes issues when we merge in the integration tests

File: src/main/ipv-stub/ipv-authorize.ts [hunk 2]
```diff
@@ -128,33 +128,47 @@ async function post(
...
+const mapFormToUserIdentity = (form: { [k: string]: string }): UserIdentity => {
+  const userIdentity: Record<string, unknown> = {};
+  if (!form["identity_claim"] || form["identity_claim"].trim().length === 0) {
+    throw new CodedError(
+      400,
+      "Invalid Request: Core Identity Claim is required"
+    );
+  }
+
+  //TODO: Should this be b64 encoded?
```
Comment (BenjaminWCO) [inline src/main/ipv-stub/ipv-authorize.ts:140]: intentional?

**Review by BenjaminWCO — APPROVED** [review]

File: src/main/ipv-stub/ipv-authorize.ts (hunk 2 above)
Comment (Ryan-Andrews99) [reply src/main/ipv-stub/ipv-authorize.ts:140]: SHould've been removed, will remove

Comment (Ryan-Andrews99) [reply src/main/ipv-stub/ipv-authorize.ts]: Removed

**Review by BenjaminWCO — APPROVED** [review]

**Pull request merged**